### DISCUSSION TITLE
Launch error

### DISCUSSION THREAD

**Discussion by tnkeo** [bonsai-rx/bonsai#1930] [General]
![Bonsai Screenshot](https://github.com/user-attachments/assets/62ddd9a4-3378-422d-b6ec-ea71ba781d3b)

Hello,

This might be a dumb question, but does anyone know why I constantly need to repair Bonsai every time I launch it?

**Comment by bruno-f-cruz** [comment]
Are you sure you are running the `bonsai.exe` executable and not the setup?
